client/trino-client/src/main/java/io/trino/client/uri/TrinoUri.java

```diff
@@ -531,6 +538,12 @@ public boolean isAssumeLiteralUnderscoreInMetadataCallsForNonConformingClients()
         return assumeLiteralUnderscoreInMetadataCallsForNonConformingClients.orElse(false);
     }
 
+    public ZoneId getTimeZone()
+            throws IllegalArgumentException
```

Comment:
Remove, as this is an unchecked exception (and I don't think it's actually thrown here)